Makefile

```diff
@@ -22,6 +23,11 @@ $(BIN)-native: $(BIN).c
 	$(Q)$(CC) $(CFLAGS) -o $@ $< \
 	    -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast -Wno-format \
 	    -ldl
+
+$(PEEP): $(PEEP).c
```

Comment:
How about renaming from `$(PEEP)` to `$(OPT)` since it is meant to be an optimizer?

Reply:
Yes.  Any changes needed are acceptable.  Are you able to edit my pull request branch directly?  I don't know if there are other errors in my make changes.

PS the merge of branch master into squint seemed to delete tests/bezier.c on my end.

Reply:
Yes, I removed `tests/bezier.c` in previous commit because of simplication.

Reply:
The peephole optimizations in Bezier were very pronounced, but I agree it was not necessary.

Reply:
PEEP is squint, the peephole optimizer executable.  $(BIN)-opt is the result of applying squint to amacc.  I am afraid that people would confuse OPT as an optimized executable of amacc rather than the peephole optimizer executable.  I used $(BIN)-opt to be consistent with $(BIN)-native that already appears in the makefile.

Reply:
Can you plese help with the make?  I can't test it at all in my environment, and I am not great at make files to begin with.